docs/working-model/organizational-structure/catena-x/expert-group.md

```diff
@@ -4,53 +4,100 @@ sidebar_position: 3
+Once an Expert Group delivered its dedicated topic, a regulated closing takes place. The Committee validates the results of the Expert Group as after the closure of the Expert Group, the responsibility for the topic transitions to the Committee. When the results are validated, the Expert Group is then officially discharged and closed by Committee.
 
 A final documentation of the work results on the digital collaboration platform is mandatory, and every Expert Group member is required to contribute to this. The documentation is to be made accessible to every member in coordination with the office.
 
-## Expert Group roles
+## Roles and Responsibilities of Expert Group
 
-### Lead and Co-Lead
+### Expert Group Leads
 
-Each Expert Group is required to elect both a Lead and a Co-Lead. These roles serve as the primary points of contact for all inquiries and requests directed towards the Committee.
+#### Election and Contact Points
 
-The Leads are tasked with ensuring that the Expert Group effectively delivers on the product strategy. They have the authority, where they deem appropriate, to delegate task execution to other Expert Group members or create additional roles within the group.
+Each Expert Group is required to elect both a Lead and a Co-Lead.
+These roles serve as the primary points of contact for all inquiries and requests directed towards the Committee.
 
-The Leads of the Working Group assumes the responsibility of monitoring and ensuring that members fulfill their time commitments effectively. Should a member consistently fail to meet their obligations, the Lead has the authority to escalate the issue to the primary contact within the member company.
+#### Ensuring Effective Delivery
 
-The Lead roles are designed to be "first among equals," responsible for maintaining cohesion and coordination within the Expert Group. They ensure that all efforts align with the Committee's objectives and facilitate efficient progress towards achieving them.
+The Leads are tasked with ensuring that the Expert Group effectively delivers on the product strategy.
+They have the authority, where they deem appropriate, to delegate task execution to other Expert Group members or create additional roles within the group.
+
+#### Monitoring and Escalation
+
+The Leads of the Expert Group assume the responsibility of monitoring and ensuring that members fulfill their time commitments effectively.
+Should a member consistently fail to meet their obligations, the Lead has the authority to escalate the issue to the primary contact within the member company.
+
+### Cohesion and Coordination
+
+The Lead roles are designed to be "first among equals," responsible for maintaining cohesion and coordination within the Expert Group.
+They ensure that all efforts align with the Committee's objectives and facilitate efficient progress towards achieving them.
+
+### Expert Group Members
```

Comment:
All points are valid but I think it is too long. Descriptions in the working model should be short. Also I hope that some of the responsibilities (joining meetings etc.) should be expected.  Otherwise I would put them higher up in the page hierachy as they apply also for other association bodies.

Reply:
i thought this structures provides an easy readability but we could also condense it like this 

"The Leads serve as the primary points of contact, ensuring the effective delivery of the product strategy by the Expert Group. They have the authority to delegate tasks and create additional roles as needed. The Leads are also responsible for monitoring member participation, escalating issues with underperforming members, and maintaining cohesion and coordination within the Expert Group to align with the Committee's objectives"